proto/storage/process_listening_on_port.proto

```diff
@@ -15,3 +17,17 @@ message ProcessListeningOnPort {
     ProcessIndicatorUniqueKey process = 3;
     google.protobuf.Timestamp close_timestamp = 4;
 }
+
+message ProcessListeningOnPortStorage {
+    // Ideally it has to be GENERATED ALWAYS AS IDENTITY, which will make it a
```

Comment:
If we do wind up going with a custom store, we could probably justify a custom schema to create a PK how you would like.

Reply:
Yep, that definitely makes sense, and PKs are in fast internal details -- ideally the app doesn't need to know that much about nor even generate them.